libs/wingc/src/ast.rs

```diff
@@ -140,19 +141,10 @@ pub enum TypeAnnotationKind {
 	MutMap(Box<TypeAnnotation>),
 	Set(Box<TypeAnnotation>),
 	MutSet(Box<TypeAnnotation>),
-	Function(FunctionTypeAnnotation),
```

Comment:
What does the VS code extension show you if you hover over a variable/field with a function type, like "fn" in the following example?

```js
class A {
  fn: inflight (num): str;
}
```

Reply:
It shows `p1` as the name of the argument. Not ideal, but good enough I think.